Neo4j.Driver/Neo4j.Driver/Internal/Routing/LoadBalancer.cs

```diff
@@ -190,6 +197,13 @@ internal IRoutingTable UpdateRoutingTable(Func<IConnection, IRoutingTable> redis
             }
         }
 
+        private void EnsureInitialRouter()
+        {
+            var ips = _seed.ToIps();
```

Comment:
The seed will be init by `ConnectionSettings.InitialServerUri`, where it will ensure no null value is provided. Though there is another constructor for testing that bypass init of `_seed`.